### PR TITLE
Use upstream repo for golang setup

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: vm-setup/requirements.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-- src: https://github.com/metal3-io/ansible-role-golang.git
-  verison: master
+- src: https://github.com/fubarhouse/ansible-role-golang/ansible-role-golang.git
```
Comment (stbenjam) [inline vm-setup/requirements.yml]: Shouldn't this be https://github.com/fubarhouse/ansible-role-golang.git?

Comment (karmab) [reply vm-setup/requirements.yml]: fixed

**Pull request merged**